src/tsdf/tsdfmetadata.py

```diff
@@ -48,11 +50,11 @@ class TSDFMetadata:
 
     file_dir_path: str
     """ A reference to the directory path, so we don't need it again when reading associated binary files. """
-    metadata_file_name: str
+    metadata_file_name: str #TODO: do we need this?? / is it used?
```

Comment:
This is only used when we want to override the existing metadata file by updating the corresponding tsdf object, but that might be unnecessary.

Reply:
So when saving existing metadata at a new location, it is not needed to update this, right?

Reply:
As far as I know, when saving to a new location the location is expected to be explicitly specified when writing to the disc (this is more of an implicit location tag)